app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user, optional: true
+  belongs_to :post, optional: true
```

Comment:
- [ ] 
- [ ] Again, as per the given _ERD_, establishing the _Like_ **association** should be such that _Like_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'

  Here also non of the associations established with the like model are optional.